content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -97,13 +133,11 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
           **Tip:** The default build includes debugging symbols, which can be useful for development but are inefficient in production. To build `rippled` for use on production servers, add the `-DCMAKE_BUILD_TYPE=Release` flag when running the `cmake` command.
 
-      2. Build the `rippled` binary executable. Replace `<number of parallel jobs>` with the number of jobs to run in parallel. Choose this value based on the number of CPU cores you want to use for building.
+      2. Build the `rippled` binary executable.
 
-              cmake --build . -- -j 4
```

Comment:
I removed the `-- -j 4` instructions b/c the build hung at 83% repeatedly when I included them. In the past, these instructions worked fine.

Reply:
as long as we are recommending cmake 3.13, then we can suggest `cmake --build . --parallel` ...the `--parallel` option is new and it will figure out what to do for the underlying generator instead of having to pass the makefile specific `-j`.

Reply:
also, I'm not sure why `-j 4` hangs for you..we should try to figure that out, although I'm curious if `--parallel` shows the same behavior.

Reply:
Discused with Mike. `--parallel` resulted in `virtual memory exhausted: Cannot allocate memory` messages for my build, so leaving all option commands out for now.